build/charts/antrea/values.yaml

```diff
@@ -248,6 +248,9 @@ controller:
     resources:
       requests:
         cpu: "200m"
+  # Exporting antrea controller service via nodeport
+  nodeport:
+    port: 0
```

Comment:
this should be moved up next to `apiPort` for clarity.

I don't understand the nexting. Why not just:
```
apiNodePort: 0
```

Reply:
Updated it.